extensions/ethereum-utils/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "ethereum-utils",
+  "title": "Ethereum Utils — EVM Development",
+  "description": "Collection of utils for Ethereum and EVM-compatible chain development",
+  "icon": "icon.png",
+  "author": "destiner",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "convert-decimals",
+      "title": "Convert Decimal Units",
+      "description": "Converts decimal unit values",
```

Comment:
```suggestion
      "description": "Converts decimal unit values",
      "subtitle": "Ethereum Utils"
```

Reply:
@mattisssa thanks for suggestion, I agree. I used "Ethereum (EVM) Utils" to allow folks to use "EVM" keyword as well.